presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -92,6 +92,11 @@ Because an ``OutOfMemoryError`` typically leaves the JVM in an
 inconsistent state, we write a heap dump, for debugging, and forcibly
 terminate the process when this occurs.
 
+Some security standards restrict the ``/tmp`` folder usage. This can prevent
```

Comment:
Let's rephrase a bit
```
The temporary directory used by the JVM must allow execution of code. Specifically,
the mount must not have the ``noexec`` flag. The default ``/tmp`` directory is
mounted with this flag in some installations, which will prevent Presto from
starting. You can workaround this by overriding the temporary directory by adding
``-Djava.io.tmpdir=/path/to/other/tmpdir`` to the list of JVM options.

Reply:
Applied this with removal of future usage